listeners/steps/articles.py

```diff
@@ -27,26 +27,18 @@ class Article:
     content: str
 
     def __post_init__(self):
-        self.publishedAt = datetime.fromisoformat(self.publishedAt[:-1]).replace(
-            tzinfo=pytz.timezone("UTC")
-        )
+        self.publishedAt = datetime.fromisoformat(self.publishedAt[:-1]).replace(tzinfo=pytz.timezone("UTC"))
 
 
-def fetch_articles(
-    api_key: str, query: str, num_articles: int = 3
-) -> List[Article]:
+def fetch_articles(api_key: str, query: str, num_articles: int = 3) -> List[Article]:
```

Comment:
These functions are not Bolt listeners. How about moving these to `utils` (new directory)? Also, having a class like `NewsFetcher` (or something like that) that has these methods instead? With this way, you can initialize news_api_key in app.py and pass a `NewsFetcher` instance with the token to `register_listeners` method.